REFERENCES.md

```diff
@@ -0,0 +1,183 @@
+# Scientific Articles
+We encourage you to also check out work by the group behind 
+GluonTS. They are grouped according to topic and ordered 
+chronographically.
+
+### Methods
```

Comment:
Shouldn't there a H2 heading in-between?

Reply:
Yes. Added this.